src/AVKit/AVPlayerViewController.cs

```diff
@@ -22,7 +22,7 @@ partial class AVPlayerViewController {
 				return AVPlayerLayer.KeyToEnum (WeakVideoGravity);
 			}
 			set {
-				WeakVideoGravity = AVPlayerLayer.EnumToKey (value);
+				WeakVideoGravity = AVPlayerLayer.EnumToKey (value)!;
```

Comment:
Again, I dislike the use of ! here, can WeakBideoGreavity be nullable?

Reply:
Similar story as the WeakExternalPlaybackVideoGravity where it comes from avfoundation.cs